test/sync/src/androidTest/kotlin/io/realm/test/shared/SyncedRealmTests.kt

```diff
@@ -93,6 +94,7 @@ class SyncedRealmTests {
     }
 
     @Test
+    @Ignore // FIXME Until https://github.com/realm/realm-kotlin/issues/555 is fixed
```

Comment:
This seem quite essential for our sync verification test. If we ignore this then we have no sync test. Is it only an issue with iOS, since it hasn't been an issue before? If so, then we should maybe move the test out of `shared` and include it on the platform where it works, or can we just somehow ignore in on iOS

Reply:
As described in the [issue](https://github.com/realm/realm-kotlin/issues/555) it's not related to iOS only (experienced on Android & macOS) so fixing the issue is high priority IMO

Reply:
Yes, then it sounds like we should jump on it. 

I don't think I have seen it in any of my PR builds yet. Have we any idea about when it started or has it always been there?

Reply:
I re-enabled the ios test